ecommerce-ios/Application/Constants/Constants.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Constants.swift
+//  ecommerce-ios
+//
+//  Created by Su T. Nguyen on 13/05/2021.
+//
+
+import SwiftUI
+
+enum Constants { }
+
+extension Constants {
+
+    enum TabBar: Int, CaseIterable {
+        case home, search, myCart, profile
+
+        var highlightedImage: Image {
+            switch self {
+            case .home: return Image("tabbar/home-fill")
+            case .search: return Image("tabbar/search-fill")
+            case .myCart: return Image("tabbar/cart-fill")
+            case .profile: return Image("tabbar/profile-fill")
+            }
+        }
+
+        var image: Image {
+            switch self {
+            case .home: return Image("tabbar/home")
+            case .search: return Image("tabbar/search")
+            case .myCart: return Image("tabbar/cart")
+            case .profile: return Image("tabbar/profile")
+            }
```

Comment:
I think you can keep the code clean by defining a variable `name`

```suggestion
        var name: String {
            switch self {
            case .home: return "home"
            case .search: return "search"
            case .myCart: return "cart"
            case .profile: return "profile"
            }
        }
 
        var highlightedImage: Image {
            return Image("tabbar/\(name)-fill")
        }

        var image: Image {
            return Image("tabbar/\(name)")
        }
```